get-started/2-install.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: page
-title: Flutter installation & setup
-permalink: /setup/
+title: "Get Started: Install Flutter"
+permalink: /get-started/install/
```

Comment:
Can we add a link to test-drive at the end of this file as well? I know in theory people will go to `setup-<platform>` and continue to `test-drive` from there, but I think there'll inevitably be people who will expect a "next step" link here as well.

Reply:
Done